### PR TITLE
Introduce ineffassign, golint and misspell

### PR DESCRIPTION
**Pull request by JulienBalestra**
### What does this PR do?

Improve the project by using standard tools.

### CODE REVIEW

File: pkg/setup/getter.go [hunk 1]
```diff
@@ -56,38 +65,47 @@ func (e *Environment) GetStaticPodPaths() ([]string, error) {
...
 func (e *Environment) IsWaitingKubeletGC() bool {
 	return e.drainOptions.KubeletGC
 }
 
+// IsSkippingStop  is a getter over the drainOptions
 func (e *Environment) IsSkippingStop() bool {
 	return e.drainOptions.None
 }
 
+// IsCleaningIptables  is a getter over the drainOptions
```
Comment (kindermoumoute) [inline pkg/setup/getter.go]: double space

File: pkg/setup/getter.go [hunk 2]
```diff
@@ -56,38 +65,47 @@ func (e *Environment) GetStaticPodPaths() ([]string, error) {
...
 func (e *Environment) IsDrainingPods() bool {
 	return e.drainOptions.Pods
 }
 
+// IsWaitingKubeletGC is a getter over the drainOptions
 func (e *Environment) IsWaitingKubeletGC() bool {
 	return e.drainOptions.KubeletGC
 }
 
+// IsSkippingStop  is a getter over the drainOptions
```
Comment (kindermoumoute) [inline pkg/setup/getter.go]: double space

File: pkg/options/drain.go [hunk 3]
```diff
@@ -14,18 +14,23 @@ import (
 	"strings"
 )
 
+// Drain is a convenient key / value bool struct to store which component
+//// should be drain
```
Comment (kindermoumoute) [inline pkg/options/drain.go]: double comment?

File: pkg/run/state.go [hunk 4]
```diff
@@ -5,6 +5,7 @@ import (
 	"sync"
 )
 
+// State keeps track of the current stats
 type State struct {
```
Comment (kindermoumoute) [inline pkg/run/state.go]: Should this struct and its methods be public?

Comment (JulienBalestra) [reply pkg/run/state.go]: That's an interesting question :+1: 
I planned to move it in a dedicated package as you might observe in #57

**Pull request merged**